modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -21,59 +47,32 @@ First, xref:maintenance/backup.adoc[backup] the following items:
 [source,console]
 ----
 # This example assumes Ubuntu Linux and MariaDB
-cp -rv /var/www/owncloud /opt/backup/owncloud && mysqldump <db_name> > /opt/backup/backup-file.sql
+mkdir /opt/backup/
+cp -rv /var/www/owncloud /opt/backup/owncloud
+mysqldump -u<username> -p<password> <databasename> > <ownCloud-Versio-Dump.sql>
 ----
 
-[[review-third-party-apps]]
 == Review Third-Party Apps
 
-Review any installed third-party apps for compatibility with the new ownCloud release. 
+Review any installed third-party apps for compatibility with the new ownCloud release.
 You can find a list of all third-party apps by running the following command, from the root directory of your installation.
 
 [source,console]
 ----
 sudo -u www-data php occ app:list --shipped=false
```

Comment:
This statement and command is **WRONG**
`--shipped=false` does NOT do what you think, the naming is a pitfall. The term shipped is misused. It means that an app can/cant be disabled but not that it is part of the tar file.
There is a core issue https://github.com/owncloud/core/issues/32048 and a possible base solution which can be enriched with additional PR´s https://github.com/owncloud/core/pull/34563